lib/layer.js

```diff
@@ -2,31 +2,62 @@ const BbPromise = require('bluebird');
 const fse = require('fs-extra');
 const path = require('path');
 const JSZip = require('jszip');
-const { writeZip, addTree } = require('./zipTree');
+const {
+  writeZip,
+  addTree
+} = require('./zipTree');
 
 BbPromise.promisifyAll(fse);
 
+/**
+ * Get the paths for the compatible runtimes of the layer
+ * @param {string[]} list of runtime paths
+ */
+function getRunTimeBuildPaths() {
+  const runtimepaths = {
+    'python2.7': 'python',
+    'python3.6': path.join('python', 'lib', 'python3.6', 'site-packages'),
```

Comment:
is `/opt/python` not in `sys.path` on python 3.6 and 3.7? I thought it was.

Reply:
It may be. I was going off of this document here
https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html

So it may just be sufficient to always put it in `/opt/python`, which would be great so that the requirements don't have to be put in multiple locations for multiple runtimes which could make the layer quite large. I'll do some testing with that to verify.

Reply:
The way I read that, all have `python`. but yeah, it's worth testing :+1:

Reply:
Yep, it works just fine to put it in `/opt/python`.